pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -374,13 +374,20 @@ let
       sha256 =
         (
           if cudaSupport then
-            { x86_64-linux = "sha256-vUoAPkYKEnHkV4fw6BI0mCeuP2e8BMCJnVuZMm9LwSA="; }
+            { x86_64-linux = "sha256-Uf0VMRE0jgaWEYiuphWkWloZ5jMeqaWBl3lSvk2y1HI="; }
           else
             {
-              x86_64-linux = "sha256-R5Bm+0GYN1zJ1aEUBW76907MxYKAIawHHJoIb1RdsKE=";
-              aarch64-linux = "sha256-P5JEmJljN1DeRA0dNkzyosKzRnJH+5SD2aWdV5JsoiY=";
+              x86_64-linux = "";
+              aarch64-linux = "";
             }
         ).${effectiveStdenv.system} or (throw "jaxlib: unsupported system: ${effectiveStdenv.system}");
+
+        # Non-reproducible fetch https://github.com/NixOS/nixpkgs/issues/321920#issuecomment-2184940546
+        preInstall = ''
+          cat << \EOF > "$bazelOut/external/go_sdk/versions.json"
```

Comment:
Note: the file has to exist, otherwise bazel-build will attempt a re-download.
Note: the original file looks something like

```
[
 {
  "version": "go1.23rc1",
  "stable": false,
  "files": [
   {
    "filename": "go1.23rc1.src.tar.gz",
    "os": "",
    "arch": "",
    "version": "go1.23rc1",
    "sha256": "6e9c4765872808663ccf0b937e5637c7df67146943fa6e8909a81a0d129c8045",
    "size": 28156320,
    "kind": "source"
   },
   {
    "filename": "go1.23rc1.aix-ppc64.tar.gz",
    "os": "aix",
    "arch": "ppc64",
    "version": "go1.23rc1",
    "sha256": "d0cfbc6fda8a6fa993e24b42ebe77cd1109fb7200cb315a90ce72d75c78075fc",
    "size": 97662103,
    "kind": "archive"
   },
   {
    "filename": "go1.23rc1.darwin-amd64.tar.gz",
```

Reply:
alternativley (needs findutils and moreutils)
```
find -name "*.json" -exec sh -c "jq --sort-keys < {} | sponge {}" \;
```

Reply:
Not sure we can do that to all of `$bazelOut/`, but at least to `$bazelOut/external/go_sdk`? Ideally, I suppose, we'd only leave the bare minimum unmasked, but let's start small:)